### DISCUSSION TITLE
PNPM support

### DISCUSSION THREAD

**Discussion by Reicr** [aws/jsii#3635] [Q&A]
Is there a way to make JSII-pacmak work with PNPM and PNPM workspaces?
Right now, packages are not build as intended and none of the PNPM configuration options helped to fix this.

**Comment by agdimech** [comment]
Once this is published you can configure the pack command to use `pnpm pack`.

https://github.com/aws/jsii/commit/6b2fd186bd77416a9b341c5544a101f50793076d
